modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/GranuleDescriptor.java

```diff
@@ -1372,7 +1373,13 @@ public GranuleLoadingResult loadRaster(
                     if (!transformed.intersects(pi.getBounds())) {
                         return null;
                     }
-                    pi.setProperty("ROI", transformed);
+                    // For some reason the "ROI" property is lost when getting the rendering
```

Comment:
Would it make sense to set it in both, wrapper and rendering? (belt and suspenders)